appveyor.yml

```diff
@@ -1,14 +1,24 @@
 version: '{build}'
+
 skip_tags: true
 clone_depth: 10
+
 environment:
+  M2_HOME: C:\maven\apache-maven-3.2.5
```

Comment:
The M2_HOME variable is not needed nor should it be used.

Reply:
@khmarbaise Actually the AppVeyor build was failing because that wasn't defined! See for example the latest build of `master` https://ci.appveyor.com/project/khmarbaise/appassembler/build/job/ptb5so3wmuiohdsb

Reply:
If that's the case they use an older version and have defined the `M2_HOME`. Apart from that we should use a more recent version of Maven 3.3.9+

Reply:
Version 3.2.5 is the last version that is compatible with JDK6. So I guess it depends on if you want to preserve JDK 6 CI testing or not?

Reply:
Version 3.2.5 is the last version of Maven which needs at minimum JDK6 but runs on JDK6+...